packages/common/src/regex.js

```diff
@@ -0,0 +1,11 @@
+// https://github.com/ideal-postcodes/postcode MIT
+
+/**
+ * Tests for a valid postcode
+ */
+module.exports.fullPostcodeRegex = /^[a-z]{1,2}\d[a-z\d]?\s*\d[a-z]{2}$/i;
```

Comment:
we need to handle ignoring whitespace in postcodes with the sql query at some point

Reply:
Do we have a bug / feat ticket for this?

Reply:
I've reverted the postcode regex to enforce spaces for now and will get a ticket in